logging.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/juju/ansiterm"
+	"github.com/juju/loggo"
+	"github.com/juju/loggo/loggocolor"
+)
+
+type colorWriter struct {
+	writer *ansiterm.Writer
+}
+
+// NewColorWriter will write out colored severity levels if the writer is
+// outputting to a terminal.
+func NewColorWriter(writer io.Writer) loggo.Writer {
+	return &colorWriter{ansiterm.NewWriter(writer)}
+}
+
+// Write implements Writer.
+//   WARNING The message...
```

Comment:
I am not sure what your intent here is... :) Is the comment showing an example output? if yes, then maybe we should also say something like:
```
Output message is prefixed with log level, i.e.
    <LOG LEVEL> <MESSAGE>
for example,
    WARNING The message
```

Reply:
Oh yup - that was copied from loggo and hacked a bit.